includes/blocks/class-sensei-block-contact-teacher.php

```diff
@@ -33,6 +33,11 @@ public function register_block() {
 				'render_callback' => [ $this, 'render_contact_teacher_block' ],
 			]
 		);
+
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Arguments used for comparison.
```

Comment:
Not critical, but WDYT to handle it separately since it's not related to the block register?

Reply:
Fixed in 606e9858447af0f98a6e498c52c630473c92f4f4